### ISSUE TITLE
[Bug] themes cause an error when being applied

### ISSUE THREAD

**Issue by LillyCookie** [t1m0thyj/WinDynamicDesktop#434]
I have the issue that some themes do not work and spit an error. I guess it's only the Author problem but I really like his themes and can't use them is there a way to fix it and if so can I get some help?

Mostly these themes I tested but I assume all from Aron do not work:
![image](https://user-images.githubusercontent.com/42301258/164511219-3140b35b-bf44-4f22-9402-3e0f71586848.png)
[Mountains #2 by Aron](https://www.mediafire.com/file/we9omzdnasbdudo/Mountains+#2.ddw/file)

![image](https://user-images.githubusercontent.com/42301258/164511303-e4dd8579-85db-4e81-98c1-846ebf4905a5.png)
[Mountains #3 by Aron](https://www.mediafire.com/file/mtwjn9bitci2p1w/Mountains+#3.ddw/file)

![image](https://user-images.githubusercontent.com/42301258/164511660-ff46ab46-dd47-4b52-b8b6-f78fbe361213.png)
[Mountains #5 by Aron](https://www.mediafire.com/file/wfmawsyz569brx7/Mountains_%25235.ddw/file#5.ddw/file)

- WDD Version: WinDynamicDesktop 5.0.1 (UWP)
- OS Version: 21H2 (OS Build 22000.613)

Steps to Reproduce:

1. double click ~.ddw file and in Select Theme click apply.
![image](https://user-images.githubusercontent.com/42301258/164510105-0818c15c-7e04-4774-b0ad-38f288cb2148.png)

2. software would display an error popup
![image](https://user-images.githubusercontent.com/42301258/164509658-7858c79d-f3cb-41ac-9056-adbd52f73152.png)

Are you using the Microsoft Store version of WDD?: `Yes`

Log File:

```
[2022-04-21 15:50:31.468] System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.AddItems(String previewName, String[] items, DateTime[] imageTimes)
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   at WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   at WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   at System.Windows.Forms.ListView.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)

[2022-04-21 15:50:38.922] System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.AddItems(String previewName, String[] items, DateTime[] imageTimes)
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   at WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   at WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   at System.Windows.Forms.ListView.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)

[2022-04-21 15:50:54.786] System.IndexOutOfRangeException: Index was outside the bounds of the array.
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.AddItems(String previewName, String[] items, DateTime[] imageTimes)
   at WinDynamicDesktop.WPF.ThemePreviewerViewModel.PreviewTheme(ThemeConfig theme)
   at WinDynamicDesktop.ThemeDialog.UpdateSelectedItem()
   at WinDynamicDesktop.ThemeDialog.listView1_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ListView.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ListView.WmReflectNotify(Message& m)
   at System.Windows.Forms.ListView.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)

[2022-04-21 16:11:12.719] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:13:12.310] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:13:30.293] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:17:25.391] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:20:59.468] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:22:34.449] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:31:44.985] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:41:13.184] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)

[2022-04-21 16:44:12.693] System.IO.FileNotFoundException: The system cannot find the file specified. (0x80070002)
   at WinDynamicDesktop.UwpHelper.SetWallpaper(String imagePath, Int32 displayIndex)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)


```

**Comment by table9gaming** [comment]
I am having this same error, it doesn't make any sense as to which computer it is happenning to.

I can't make any sense of the error log either, mine looks pretty much exactly the same other than username of course.

**Comment by table9gaming** [comment]
In my testing this is happening to ALL wallpapers, this happens regardless of newer or older verison (tried several older builds) 

on the computers that they work on, they still work, but for others no matter what version it "installs" but as soon as you try to apply or download a theme it errors out and forces me to go to task manager and kill the process.

**Comment by t1m0thyj** [comment]
Fixed in WinDD 5.4.0
